### PR TITLE
 Completed Order: Hides Billing Info if Enable Billing checkbox is unchecked

### LINKED ISSUE #2206: Completed Order: Billing info is visible even when the Enable Billing checkbox is unchecked
**Describe the bug**

When someone orders a ticket and keeps `Enable Billing` unchecked it shouldn't display on the `Completed Order` page. `Billing Info` must be hidden in this case
An attribute `isBillingEnabled` is added to the orders object here - https://github.com/fossasia/open-event-server/pull/5620

**To Reproduce**
1. Order a ticket with `Enabled Billing` unchecked
2. Proceed to checkout
3. See error

**Expected behavior**

`Billing Info` should be hidden

**Screenshots**

screenshot_2019-02-18 completed order -9ef780d3-b26d-4b16-bbf0-af5d71eaada8 orders open event

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu 18.04
 - Browser [e.g. chrome, safari] Firefox

**Additional context**

Working on it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
When someone orders a ticket and keeps `Enable Billing` unchecked it shouldn't display on the `Completed Order` page. `Billing Info` must be hidden in this case

#### Changes proposed in this pull request:

- Hides Billing Info if Enable Billing checkbox is unchecked

Fixes #2206

### CODE REVIEW

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 I am working on adding `isBillingEnabled` in `order-form`. Everything is working and value is also updating properly on frontend. But when a PATCH request is sent to server, it returns an error. It is basically FORBIDDEN. I'm unable to understand why. Please help

`Error: Ember Data Request PATCH http://127.0.0.1:5000/v1/orders/c2257bcf-b2b5-47fe-b70b-86e18cc47bc9 returned a 403 Payload (Empty Content-Type) [object Object] Oops something went wrong. Please try again`

**Comment by CosmicCoder96** [comment]
Post the body of the PATCH requested generated by your code.

**Comment by ritikamotwani** [comment]
@shreyanshdwivedi any updates?

**Comment by shreyanshdwivedi** [comment]
@ritikamotwani @CosmicCoder96 PFA the body of PATCH request
screenshot from 2019-03-08 17-33-49
screenshot from 2019-03-08 17-33-32

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 I've fixed the issue. PR on server side - https://github.com/fossasia/open-event-server/pull/5683
Screenshots of UI change made by this PR - 
Screenshot_2019-03-14 Completed Order -9ef780d3-b26d-4b16-bbf0-af5d71eaada8 Orders Open Event
Screenshot_2019-03-14 Completed Order -9891defc-3d1b-4bea-a08e-957bcb19949b Orders Open Event

Please review :)

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 you've merged the corresponding PR on server. Please review this too

File: app/templates/components/forms/orders/order-form.hbs [hunk 1]
```diff
@@ -69,10 +69,10 @@
       {{/each}}
       {{#if isPaidOrder}}
         <div class="field">
-          {{ui-checkbox label=(t 'Enable Billing Information') checked=enablebilling
-                          onChange=(action (mut enablebilling))}}
+          {{ui-checkbox label=(t 'Enable Billing Information') id='isBillingEnabled' checked=data.isBillingEnabled
```
Comment (CosmicCoder96) [inline app/templates/components/forms/orders/order-form.hbs]: why assign it an `id`, not required.

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 I've incorporated the changes. Please review

**Pull request merged**